package.json

```diff
@@ -36,11 +35,10 @@
     "lint:ts:fix": "tslint -c tslint.json -p . --fix",
     "ganache": "ganache-cli --networkId ${npm_package_config_ganacheNetworkID} --port ${npm_package_config_ganachePort} --gasLimit ${npm_package_config_ganacheGasLimit} --gasPrice ${npm_package_config_ganacheGasPrice} --defaultBalanceEther ${npm_package_config_etherBalance} --deterministic --mnemonic \"${npm_package_config_mnemonic}\" ${npm_package_config_extra}",
     "ganache:verbose": "ganache-cli --networkId ${npm_package_config_ganacheNetworkID} --verbose --port ${npm_package_config_ganachePort} --gasLimit ${npm_package_config_ganacheGasLimit} --gasPrice ${npm_package_config_ganacheGasPrice} --defaultBalanceEther ${npm_package_config_etherBalance} --deterministic --mnemonic \"${npm_package_config_mnemonic}\" ${npm_package_config_extra}",
-    "ganache:stop": "ps aux | grep ganache-cli | grep -v grep | awk '{print $2}' | xargs kill -9",
-    "console": "truffle console"
+    "ganache:stop": "ps aux | grep ganache-cli | grep -v grep | awk '{print $2}' | xargs kill -9"
   },
   "config": {
-    "mnemonic": "dose weasel clever culture letter volume endorse used harvest ripple circle install",
+    "mnemonic": "test test test test test test test test test test test junk",
```

Comment:
Maybe we should leave the previous one, but either way seems fine.

Reply:
yea I do like the latter since it is self-explanatory that it's a test mnemonic